bin/Replication/setup-test-replication.sql

```diff
@@ -0,0 +1,40 @@
+-- Set up distribution
```

Comment:
This file should be added to dataplat/appveyor-lab and referenced from that repository. You can include checking this repo out in the GH action.

Reply:
No longer required - will remove file.